src/x.hd.wallet.api.crypto.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from 'libsodium-wrappers-sumo';
 import * as msgpack from "algo-msgpack-with-bigint"
 import Ajv from "ajv"
-import { deriveChildNodePrivate } from './bip32-ed25519';
+import { deriveChildNodePrivate } from './bip32-ed25519.js';
```

Comment:
Can't we avoid `.js` ?

Reply:
https://github.com/algorandfoundation/xHD-Wallet-API-ts/pull/8#discussion_r1763193391
@PhearZero

Reply:
@ehanoc this is considered proper ESM with TypeScript. 

Reference: https://www.typescriptlang.org/docs/handbook/modules/appendices/esm-cjs-interop.html#different-module-resolution-algorithms

Reply:
I'm pretty sure we can change the `tsconfig.json` to ensure ESM compatibility without having to declare these in the `.ts` files though. Can you check @HashMapsData2Value ?

Reply:
@ehanoc this has been resolved by using tsc-alias as part of the build procedure.